src/Tasks/AssemblyDependency/ResolveAssemblyReference.cs

```diff
@@ -2266,17 +2291,23 @@ out _copyLocalFiles
                                         useSystemRuntime = true;
                                         break;
                                     }
+                                    if (string.Equals(NETStandardAssemblyName, dependentReference.Name, StringComparison.OrdinalIgnoreCase))
+                                    {
+                                        useNetStandard = true;
+                                        break;
+                                    }
```

Comment:
where is this logic in current code base? ...and does it still reflect this logic, `break` with `useSystemRuntime` assignment without checking subsequent `dependentReferences` iteration for a dependency on `NETStandardAssemblyName` and assigning `useNetStandard`?

i.e.
In a scenario where `useSystemRuntime` is `true` and `useNetStandard` is `false` the outer scope `(!useSystemRuntime || !useNetStandard)` condition logic is `true`, with warrants entering this scope with the intent to determine whether `useNetStandard` should be `true`, right?  doesn't the `break` provide a scenario where it won't\didn't?

Reply:
potentially, this should be\have been (may be now)
```
if (!useSystemRuntime && string.Equals(SystemRuntimeAssemblyName, dependentReference.Name, StringComparison.OrdinalIgnoreCase))
{
	useSystemRuntime = true;
} 
else if (!useNetStandard && string.Equals(NETStandardAssemblyName, dependentReference.Name, StringComparison.OrdinalIgnoreCase))
{
	useNetStandard = true;
}

if (useNetStandard && useSystemRuntime)
{
	break;
}
```

Reply:
this depends on whether it's possible that `dependencyTable.References.Values` can have both, can it?